config/project/sections/faculties--a71afc7e-315c-4be0-b69b-5c70a7dc85db.yaml

```diff
@@ -0,0 +1,28 @@
+defaultPlacement: end
+enableVersioning: true
+entryTypes:
+  -
+    uid: c8c8ccd9-ebdc-48dd-bac8-029cb3d04923 # Faculty
+handle: faculties
+maxAuthors: 1
+name: Faculties
```

Comment:
Are we going with Faculty for this entry type or Faculty for the People sub-type? We could use “Departments” instead of “Faculties” here for better disambiguation.